Markdown/cmm-2.0.14/release-notes/2-restrictions.md

```diff
@@ -60,4 +60,16 @@ Errors occur due to this known issue in Kibana 7.3.0 [Kibana Issue 41567](https:
   ```
   hostname: "docker-host" and not (service: "kibana" or service: "logspout")
   ```
+
+### Importing Objects in Kibana
+Exported JSON files can only be imported with API, due to a known issue with Kibana 7.3.0:  
+When attempting to import from web UI, an error message is shown, including information of "Unsupported Media Type".
+
+* Workaround:  
+  Use following `curl` command to import the JSON data from API: 
+  ```
+  curl –XPOST “<ip-address of kibana server>:5601/api/saved_objects/_import?overwrite=true” -H "kbn-xsrf: true" --form file=@<file to be imported>
```

Comment:
Thanks for the PR, let's use the these kind of quotes: `"`

Reply:
Done